api/microsoft.js

```diff
@@ -15,5 +15,7 @@ module.exports = function(pubUrl) {
     }
   }).then(function(res) {
     return res.tags;
+  }).catch(function(err) {
+    console.log('err requesting from microsoft', err);
```

Comment:
I tried a catch block for the Microsoft API and for some reason they always sent back a non-null error, even with valid requests. Did you encounter this?